### PR TITLE
Prune BlockChaos in release-2.2

### PR DESCRIPTION
**Pull request by STRRL**
Revert "blockchaos: api definition and selector implementation (#2739)"

This reverts commit 618a4cb8e697d7545b211e0a86aa2bbdb018c22a.

### CODE REVIEW

File: compile_commands.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[
+    {
+        "command": "cc -c ./pkg/time/fakeclock/fake_clock_gettime.c -fPIE -O2 -o pkg/time/fakeclock/fake_clock_gettime.o",
```
Comment (YangKeao) [inline compile_commands.json:3]: Remove this file

File: Makefile [hunk 2]
```diff
@@ -56,7 +55,7 @@ BASIC_IMAGE_ENV=IMAGE_DEV_ENV_PROJECT=$(IMAGE_DEV_ENV_PROJECT) IMAGE_DEV_ENV_REG
 	IMAGE_BUILD_ENV_TAG=$(IMAGE_BUILD_ENV_TAG) IN_DOCKER=$(IN_DOCKER) \
 	IMAGE_TAG=$(IMAGE_TAG) IMAGE_PROJECT=$(IMAGE_PROJECT) IMAGE_REGISTRY=$(IMAGE_REGISTRY) \
 	TARGET_PLATFORM=$(TARGET_PLATFORM) \
-	GO_BUILD_CACHE=$(GO_BUILD_CACHE) YARN_BUILD_CACHE=$(YARN_BUILD_CACHE)
```
Comment (YangKeao) [inline Makefile:59]: Could we keep these configuration?

**Review by YangKeao — COMMENTED** [review]
Rest LGTM

File: Makefile (hunk 2 above)
Comment (STRRL) [reply Makefile:59]: I am not so familiar with these2 changes,maybe youcould push.to my branch directly?❤️❤️

Comment (YangKeao) [reply Makefile]: Done.

File: compile_commands.json (hunk 1 above)
Comment (STRRL) [reply compile_commands.json]: Updated! PTAL

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**